ydb/src/client_builder.rs

```diff
@@ -101,6 +102,48 @@ fn token_metadata(uri: &str, mut client_builder: ClientBuilder) -> YdbResult<Cli
     Ok(client_builder)
 }
 
+fn password(uri: &str, mut client_builder: ClientBuilder) -> YdbResult<ClientBuilder> {
+    let mut username = Option::<String>::default();
+    let mut password = Option::<String>::default();
+
+    for (key, value) in url::Url::parse(uri)?.query_pairs() {
+        match key.as_ref() {
+            "password" => {
```

Comment:
A command for detect static credentials type auth is `token_password` (or `token_static_password` in future). 
Now user must set both fields `token_password` and `password`: first for enable the handler and second for set password.

Need consistent parameter name: token_static_login and token_static_password.

Reply:
Maybe it's better to use `token_static_username` as a parameter name?